snap/hooks/configure

```diff
@@ -68,6 +68,10 @@ if [[ -z $(snapctl get kubelet.offline-mode) ]]; then
     snapctl set kubelet.offline-mode=true
 fi
 
+if [[ -z $(snapctl get docker.bridge) ]]; then
```

Comment:
For the purpose of unified style, this works. Similar comment as above applies though: `-z` would normally be used with `[` and not `[[` as it originates from the same time frame.